frontend/packages/topology/src/components/application-panel/TopologyApplicationResources.tsx

```diff
@@ -18,7 +18,9 @@ const TopologyApplicationResources: React.FC<TopologyApplicationResourcesProps>
 }) => {
   const { t } = useTranslation();
   const resourcesData = resources.reduce((acc, { resource }) => {
-    acc[resource.kind] = [...(acc[resource.kind] ? acc[resource.kind] : []), resource];
+    if (resource?.kind) {
+      acc[resource.kind] = [...(acc[resource.kind] ? acc[resource.kind] : []), resource];
+    }
```

Comment:
@vikram-raj By skipping the Helm Resource entry, will cause the sidebar to have empty resources section.

![image](https://user-images.githubusercontent.com/9964343/147732991-a9da49aa-9222-47f9-b120-82ff650fcdf2.png)

I suggest, let us destructure `resourceKind` from the props and use it as fallback if `resource.kind` is not available and pass the appropriate resource object based on the condition so it will populate the sidebar with Helm Release Object.